### DISCUSSION TITLE
Invited to copilot via email, but cannot make it work

### DISCUSSION THREAD

**Discussion by c3plan** [community/community#18828] [Copilot Conversations]
I received an email and followed the instructions, but now I'm receiving a message that I have to join a waiting list. So why did you email me for? I suppose I've done something wrong, or the invitation instructions aren't complete

**Comment by davecheney** [comment]
@c3plan I see that your account is on the waitlist. We're inviting people on a daily basis. Thank you for your patience.

**Reply by c3plan** [reply]
That's strange though, because I received an email: Welcome to GitHub Copilot Technical Preview!

The email says that I was being invited to test, and that I should follow the instructions - so I did, but it did not work. I tried through many ways, and one of them was by subscribing again.

Then it said I should post problems here, and so I did.

The error message I've been receiving is this:
[INFO] [auth] [2022-06-18T02:55:28.630Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2022-06-18T02:55:28.631Z] GitHub Copilot could not connect to server. Extension activation failed: "User not authorized"

**Reply by davecheney** [reply]
I’m sorry but your account is not active, it remains on the waitlist. We’re inviting people on a daily basis. Please be patient

**Reply by davecheney** [reply]
@natka666 yes

**Comment by davecheney** [comment]
@c3plan it looks like your account was sent an invite today and copilot is now active for you.

**Reply by c3plan** [reply]
Awesome - it's working now! Thanks

**Comment by zTrolly** [comment]
I received the same email, and I am not able to use it. Could you check for me if I'm still on the waiting list?
<img width="560" alt="Captura de Tela 2022-06-18 às 23 13 48" src="https://user-images.githubusercontent.com/61760048/174463058-197cf5d6-6219-45b2-8a3f-63aa8a8476ae.png">
